docs/screencast-compatibility.md

```diff
@@ -0,0 +1,124 @@
+# Screencast Compatibility Guide
+
+This guide contains information on applications compatible with
+xdg-desktop-portal-wlr. If screencasting works for you in some applications,
+but not others, please review this guide.
+
+## WebRTC
+
+WebRTC is a protocol and specification for real-time communication,
+but it is also a shared codebase and a set of APIs used by Firefox and Chrome
+to provide screen capture and casting functionality.
+
+WebRTC contains support for xdg-desktop-portal based screen casting, but
+requires a special build flag ```RTC_USE_PIPEWIRE=true```
+in order to be included. Some distros build their browsers with this
+flag and some do not.
+
+No matter what browser you are using, the WebRTC code will check that the
+environment variable  ```XDG_SESSION_TYPE=wayland``` is set
+in order to function.
+
+Additionally, there is a compatibility issue related to pixelformats that
+currently requires the flag ```-p BGRx``` to be added to xdg-desktop-portal-wlr
+in order to get WebRTC to correctly negotiate a pipewire stream. Support for
+additional pixel formats in WebRTC will fix this issue, but we are also
+discussing being more liberal about the formats we claim. You can read more
+about that [here](https://github.com/emersion/xdg-desktop-portal-wlr/issues/23).
```

Comment:
WDYT about adding a link to the webrtc test available on this page: https://mozilla.github.io/webrtc-landing/gum_test.html?

Reply:
That's a good idea. I'll link it before we commit.